### PR TITLE
Bit 311 increase coverage

### PR DESCRIPTION
**Pull request by shibshib**
Increase coverage of testing on Bittensor from ~60% to ~82% so far.

This PR also removes some vestigial code.

### CODE REVIEW

File: bittensor/_wandb/__init__.py [hunk 1]
```diff
@@ -60,6 +59,7 @@ def __new__(
         os.environ["WANDB_PROJECT"] = config.wandb.project if config.wandb.project != 'default' else str(cold_pubkey)[:8]
         os.environ["WANDB_TAGS"] = config.wandb.tags 
         os.environ["WANDB_RUN_GROUP"] = config.wandb.run_group if config.wandb.run_group != 'default' else str(hot_pubkey)[:8]
+        import pdb; pdb.set_trace()
```
Comment (unconst) [inline bittensor/_wandb/__init__.py]: maybe remove this

File: bittensor/utils/__init__.py [hunk 2]
```diff
@@ -39,55 +39,6 @@ def indexed_values_to_dataframe (
             dataframe.loc[idx_i] = pandas.Series( { str(prefix): value_i } )
     return dataframe
 
-def indexed_values_to_wandb(
```
Comment (unconst) [inline bittensor/utils/__init__.py:42]: Is this not used elsewhere?

File: bittensor/utils/networking.py [hunk 3]
```diff
@@ -151,29 +151,6 @@ class UPNPCException(Exception):
     """ Raised when trying to perform a port mapping on your router. """
 
 
-def upnpc_delete_port_map(external_port: int):
```
Comment (unconst) [inline bittensor/utils/networking.py:154]: Is this also not being used, im suprised.

**Review by unconst — APPROVED** [review]

**Comment by unconst** [comment]
LGTM

File: bittensor/_wandb/__init__.py (hunk 1 above)
Comment (shibshib) [reply bittensor/_wandb/__init__.py]: whoops. good catch.

File: bittensor/utils/__init__.py (hunk 2 above)
Comment (shibshib) [reply bittensor/utils/__init__.py:42]: Nope.

File: bittensor/utils/networking.py (hunk 3 above)
Comment (shibshib) [reply bittensor/utils/networking.py:154]: Nope.

**Pull request merged**